assignment1/training.js

```diff
@@ -0,0 +1,67 @@
+// Problem 1: Complete the secondLargest function which takes in an array of numbers in input and return the second biggest number in the array. (without using sort)?
+function secondLargest(array) {
+  // Write your code here
+  max=Number.MIN_SAFE_INTEGER;
+  max2=Number.MIN_SAFE_INTEGER;
+  
+  for(i=0;i<array.length;i++)
+    {
+      if(array[i]>max) max=array[i];
+      
+   }
+  for(i=0;i<array.length;i++)
+    {
+      if(array[i]>max2&&array[i]<max) max2=array[i];
+    }
+  return max2;
+}
+
+// Problem 2: Complete the calculateFrequency function that takes lowercase string as input and returns frequency of all english alphabet. (using only array, no in-built function)
+function calculateFrequency(string) {
+  // Write your code here
+   var freq={};
+    var i;
+  for(i=0;i<string.length;i++)
+    {
+      var x=string.charAt(i);
+      if(x.match(".*[a-z].*")){
+      if(freq[string.charAt(i)] ) {freq[string.charAt(i)]++;}
```

Comment:
@psinha20 Please do code indentation properly & write `freq[string.charAt(i)]++;` in new line.